### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by codedracula4ec**
adds pt-BR translation to the alpha release

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -1,5 +1,664 @@
...
+#: Dialogs/ReleaseDialog.vala:47 Dialogs/SettingsDialog.vala:217
+msgid ""
+"WARNING!\n"
+"Akira is still under development and not ready for production. Missing "
+"features, random bugs, and black holes opening in your kitchen are to be "
+"expected."
+msgstr ""
+"ATENÇÃO!\n"
+"Akira está sob desenvolvimento e não está pronto para produção. É de se "
+"esperar mal funcionamento e inexistência de funções."
```
Comment (isneezy) [inline po/pt_BR.po:63]: É de se esperar o mau funcionamento e a inexistência de algumas funcionalidades.

File: po/pt_BR.po [hunk 2]
```diff
@@ -1,5 +1,664 @@
...
+msgid "Use Symbolic Icons:"
+msgstr "Usar ícones monocromáticos:"
+
+#: Dialogs/SettingsDialog.vala:129
+msgid "Default Colors"
+msgstr "Cores padrão"
+
+#: Dialogs/SettingsDialog.vala:131
+msgid "Define the default style used when creating a new shape."
+msgstr "Define o estilo padrão ao criar uma nova forma."
```
Comment (isneezy) [inline po/pt_BR.po:132]: Define o estilo padrão a ser usado para criar uma nova forma.

**Review by isneezy — CHANGES_REQUESTED** [review]
Hi, here are some few things to hel improve your translation, I'm not sure about the glyphs, but any way thanks. It's nice to see this is attracting people who speak Portuguese like me. :smile:

File: po/pt_BR.po (hunk 1 above)
Comment (codedracula4ec) [reply po/pt_BR.po]: That makes sense. Tanks!

**Comment by Alecaddd** [comment]
@isneezy I'll wait for your approved review on this before merging.
Thank you both for the contribution!

File: po/pt_BR.po [hunk 3]
```diff
@@ -1,5 +1,664 @@
...
+msgid "Invert Panels Order:"
+msgstr "Inverter ordem dos painéis:"
+
+#: Dialogs/SettingsDialog.vala:96
+msgid "Restart application to apply this change."
+msgstr "Reinicie a aplicação para aplicar a mudança."
+
+#: Dialogs/SettingsDialog.vala:100
+msgid "ToolBar Style"
+msgstr "Etilo da barra de ferramentas"
```
Comment (isneezy) [inline po/pt_BR.po:116]: there is a typo here, it should be `Estilo`

File: po/pt_BR.po [hunk 4]
```diff
@@ -1,5 +1,664 @@
...
+msgid "Opacity"
+msgstr "Opacidade"
+
+#: Layouts/RightSideBar.vala:111
+msgid "Search Layer"
+msgstr "Buscar Camada"
+
+#: Lib/Managers/ExportManager.vala:192 Lib/Managers/ExportManager.vala:219
+msgid "Generating preview, please wait…"
+msgstr "Gerando previsualização, aguarde…"
```
Comment (isneezy) [inline po/pt_BR.po:576]: Gerando pré-visualização, aguarde por favor…

**Review by isneezy — CHANGES_REQUESTED** [review]
I found some more needles

File: po/pt_BR.po (hunk 4 above)
Comment (codedracula4ec) [reply po/pt_BR.po]: fixed on commit df424068db09ebe94af14a03d54bf94d016c2578

**Review by isneezy — APPROVED** [review]

**Pull request merged**